### DISCUSSION TITLE
What would be a good domain name for GoodJob

### DISCUSSION THREAD

**Discussion by bensheldon** [bensheldon/good_job#744] [General]
I've been thinking of having a better frontdoor for GoodJob than simply the giant GitHub Readme.

What would be a good domain name? 

- `goodjobjobs.com`
- `goodjob.io` (ugh, but the cost)

Suggestions? I like playing with https://domainr.com/

**Comment by edouard** [comment]
How about domains in [good-job](https://domainr.com/?q=good-job) — good-job.org, good-job.io are available

Or domains in [goodjobrb](https://domainr.com/?q=goodjobrb) — goodjobrb.org, goodjobrb.io are available.

I like good-job.com better. It’s read easier than goodjobrb.com, which can be scanned as “brb” 😁
